scss/title.scss

```diff
@@ -0,0 +1,17 @@
+.title {
```

Comment:
i wouldn't do a file for each component. Use styles.scss for all or maybe devide by section (header, content, footer)